### ISSUE TITLE
Official Train-Test 

### ISSUE THREAD

**Issue by qiubs** [MIT-LCP/mimic-code#770]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Multiple publications* have been using this MIMIC-III to formulate a clinical document classification task by joining clinical note text (`NOTEEVENTS.TEXT`) with ICD-9 code (`DIAGNOSES_ICD.ICD9_CODE`). The lack of official train-test splits for this task has made these results incomparable due to separate splitting schemes. This is contrary to MIMIC's objective of delivering reproducible reusable experiments for clinical notes.

The solution I propose is that I could submit a PR containing a label-extracting notebook for 50 most frequent ICD-9 labels in the `/benchmark/` directory that could designated as MIMIC-III's official train-test splits for this task. Would MIMIC's authors approve of this?

*Example Publications using ICD-9 classification:
Paper 1 name: Explainable Prediction of Medical Codes from Clinical Text
Paper link: https://arxiv.org/abs/1802.05695
Github Repo: https://github.com/jamesmullenbach/caml-mimic 

Paper 2 name: An Empirical Evaluation of Deep Learning for ICD-9 Code Assignment using MIMIC-III Clinical Notes
Paper link: https://arxiv.org/abs/1802.02311
Github Repo: https://github.com/lsy3/clinical-notes-diagnosis-dl-nlp

Paper 3 name: Natural language processing of MIMIC-III clinical notes for identifying diagnosis and procedures with neural networks
Paper link: https://arxiv.org/pdf/1912.12397
Github Repo: https://github.com/SiddharthaNuthakki/NLP-Clinical-notes-Neural-Networks-AWD-LSTM-

**Comment by alistairewj** [comment]
I agree having consistent splits would be useful across papers. Putting code to generate it in the `benchmarks` folder would work. Going even further, it would be best to publish a derived dataset on PhysioNet as well to maximize reuse. It might make more sense to establish this for MIMIC-IV though when the notes are made available!

**Comment by h4ste** [comment]
I am working on a derived dataset which will be uploaded to PhysioNet soon. I have prepared a benchmark collection from MIMIC-III notes with fixed training, development, calibration, and testing splits for phenotyping, mortality prediction, readmission, length-of-stay, and various disease staging tasks. Splits are stratified by demographics and admission information to facilitate generalization and reduce the impact of confounders.

Currently, the collection is based on MIMIC-III. Is there any estimate for when notes will become available for MIMIC-IV (on the order of weeks, months, or years)?
